From ac11cf8cfd1e0405cf1f4f7d394a552dd411dd0f Mon Sep 17 00:00:00 2001
Message-Id: <patch-v2-1.8-ac11cf8cfd1-20210910T102726Z-avarab@gmail.com>
In-Reply-To: <cover-v2-0.8-00000000000-20210910T102726Z-avarab@gmail.com>
References: <cover-0.5-00000000000-20210908T152807Z-avarab@gmail.com>
	<cover-v2-0.8-00000000000-20210910T102726Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 10 Sep 2021 12:16:51 +0200
Subject: [PATCH v2 1/8] INSTALL: don't mention the "curl" executable at all
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

In 1d53f90ed97 (The "curl" executable is no longer required,
2008-06-15) the wording for requiring curl(1) was changed to the
current "you might also want...".

Mentioning the "curl" executable at all is just confusing, someone
building git might want to use it to debug things, but they might also
just use wget(1) or some other http client. The "curl" executable has
the advantage that you might be able to e.g. reproduce a bug in git's
usage of libcurl with it, but anyone going to those extents is
unlikely to be aided by this note in INSTALL.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 INSTALL | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/INSTALL b/INSTALL
index 66389ce0591..5c6ecf03c27 100644
--- a/INSTALL
+++ b/INSTALL
@@ -139,11 +139,11 @@ Issues of note:
 	  (PPC_SHA1).
 
 	- "libcurl" library is used by git-http-fetch, git-fetch, and, if
-	  the curl version >= 7.34.0, for git-imap-send.  You might also
-	  want the "curl" executable for debugging purposes. If you do not
-	  use http:// or https:// repositories, and do not want to put
-	  patches into an IMAP mailbox, you do not have to have them
-	  (use NO_CURL).
+	  the curl version >= 7.34.0, for git-imap-send.
+
+	  If you do not use http:// or https:// repositories, and do
+	  not want to put patches into an IMAP mailbox, you do not
+	  have to have them (use NO_CURL).
 
 	- "expat" library; git-http-push uses it for remote lock
 	  management over DAV.  Similar to "curl" above, this is optional
-- 
2.33.0.867.gd5d659bf226

